graphql/handler/transport/headers.go

```diff
@@ -11,17 +11,18 @@ const (
 	acceptApplicationGraphqlResponseJson = "application/graphql-response+json"
 )
 
-func determineResponseContentType(explicitHeaders map[string][]string, r *http.Request) string {
+func determineResponseContentType(explicitHeaders map[string][]string, r *http.Request, useGrapQLResponseJsonByDefault bool) string {
```

Comment:
It seems a little odd to have the parameter order be `r *http.Request, useGrapQLResponseJsonByDefault bool` instead of `useGrapQLResponseJsonByDefault bool, r *http.Request` but I suppose it doesn't really matter.

Reply:
@StevenACoffman Thanks for your review and suggestions. Should I adjust the order accordingly?